lib/Pusher.php

```diff
@@ -330,6 +352,7 @@ public function trigger( $channels, $event, $data, $socket_id = null, $debug = f
 
 		if ( $socket_id !== null )
 		{
+			$this->validate_socket_id( $socket_id );
```

Comment:
Or:

```php
        $this->validate_socket_id( $socket_id );

...

        if ( $socket_id !== null )
        {
            $post_params[ 'socket_id' ] = $socket_id;
        }
```

Since `null` isn't an invalid socket_id. It just indicates there isn't one.